Makefile

```diff
@@ -15,18 +15,20 @@
 # Set ENV to 'YELP' if FQDN ends in '.yelpcorp.com'
 # Otherwise, set ENV to the FQDN
 ifeq ($(findstring .yelpcorp.com,$(shell hostname -f)), .yelpcorp.com)
```

Comment:
What about something like an if on whether you can `curl pypi.yelpcorp.com/simple` Would mean we depend on the user having curl installed... Maybe netcat is more ubiquitous `nc -v pypi.yelpcorp.com 443 -z`?

For example my laptop doesn't seem to have `hostname -f` with yelpcorp.com even when I'm on the Yelp network.